metrics/enterprise.go

```diff
@@ -0,0 +1,50 @@
+package metrics
+
+import (
+	"log"
+	"strconv"
+
+	"github.com/cloudbase/garm/auth"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+// CollectOrganizationMetric collects the metrics for the enterprise objects
+func (c *GarmCollector) CollectEnterpriseMetric(ch chan<- prometheus.Metric, hostname string, controllerID string) {
+	ctx := auth.GetAdminContext()
+
+	enterprises, err := c.runner.ListEnterprises(ctx)
+	if err != nil {
+		log.Printf("listing providers: %s", err)
+		// continue anyway
```

Comment:
in all likelihood the `enterprises` variable is an empty list, so you can simply `return` here and skip the rest of the code.

Reply:
found this pattern in the existing code and I just kept using it without thinking :see_no_evil:

Reply:
No worries! There are plenty of places we can improve in time.